packages/sdk/src/Portfolio/Integrations/AaveV3.ts

```diff
@@ -610,9 +610,23 @@ export async function getAvailableSupplyAmount(
   args: Viem.ContractCallParameters<{
     protocolDataProvider: Address;
     asset: Address;
+    decimals: number;
   }>,
 ) {
   const [reserveCaps, reserveData] = await Promise.all([getReserveCaps(client, args), getReserveData(client, args)]);
 
-  return reserveCaps.supplyCap - reserveData.totalAToken;
+  return reserveCaps.supplyCap * 10n ** BigInt(args.decimals) - reserveData.totalAToken;
+}
+
+export async function getAvailableVariableDebtAmount(
+  client: Client,
+  args: Viem.ContractCallParameters<{
+    protocolDataProvider: Address;
+    asset: Address;
+    decimals: number;
+  }>,
+) {
+  const [reserveCaps, reserveData] = await Promise.all([getReserveCaps(client, args), getReserveData(client, args)]);
+
+  return reserveCaps.borrowCap * 10n ** BigInt(args.decimals) - reserveData.totalVariableDebt;
```

Comment:
could be also `return parseUnits(reserveCaps.borrowCap.toString(), args.decimals) - reserveData.totalVariableDebt;`

Reply:
I had that first, but didn't like the double conversion.